composer.json

```diff
@@ -17,24 +17,24 @@
     "process-timeout": 0,
     "sort-packages": true
   },
-  "repositories": [
-    {
+  "repositories": {
+    "0": {
```

Comment:
For knowledge sharing just wondering what's the benefit of using keys here?

Reply:
no specific reason for that - composer CLI update it for me (probably keys are added by default now).